python/ray/dashboard/client/src/pages/dashboard/logical-view/Actor.tsx

```diff
@@ -181,14 +205,76 @@ class Actor extends React.Component<Props & WithStyles<typeof styles>, State> {
             {
               label: "Required resources",
               value:
-                Object.entries(actor.requiredResources).length > 0 &&
-                Object.entries(actor.requiredResources)
-                  .sort((a, b) => a[0].localeCompare(b[0]))
-                  .map(([key, value]) => `${value.toLocaleString()} ${key}`)
-                  .join(", "),
+                Object.entries(actor.requiredResources).length > 0
+                  ? Object.entries(actor.requiredResources)
+                      .sort((a, b) => a[0].localeCompare(b[0]))
+                      .map(([key, value]) => `${value.toLocaleString()} ${key}`)
+                      .join(", ")
+                  : "",
             },
           ];
 
+    // Move some fields to the back and de-prioritize them.
+    const secondaryFields = [
+      "NumObjectIdsInScope",
+      "NumLocalObjects",
+      "UsedLocalObjectMemory",
+    ];
+    const secondaryRawInformation: ActorInformation[] = [];
+    secondaryFields.forEach((fieldName) => {
+      const foundIdx = rawInformation.findIndex(
+        (info) => info.label === fieldName,
+      );
+      if (foundIdx !== -1) {
+        const foundValue = rawInformation[foundIdx];
+        rawInformation.splice(foundIdx, 1);
+        secondaryRawInformation.push(foundValue);
+      }
+    });
```

Comment:
Rather than constructing this via a process that splits the information into primary and secondary keys, how about we just directly instantiate two arrays above? Maybe something like this:

```
const information = actor.state !== -1 ? { primary: ..., secondary: ... } : { primary: ..., secondary: ... };
const informationElements = { primary: raw.primary.map(transform), secondary: raw.secondary.map(transform) };
```

(Feel free to adjust the names as required.)